packages/web-app/src/components/executionWidget/actionsFilter.tsx

```diff
@@ -27,6 +27,8 @@ export const ActionsFilter: React.FC<ActionsFilterProps> = ({action}) => {
       );
     case 'add_member':
       return <AddMemberCard action={action} />;
+    case 'credit_delegation':
+      return <></>;
```

Comment:
We should also create this card to show the action in the proosal

Reply:
Yep, i was doing that